src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -360,6 +365,23 @@ function GardenMetadata() {
                   onFileUpdated={handleOnTokenLogoUpdated}
                 />
               </div>
+              {config.garden.type === BYOT_TYPE ? (
+                <div
+                  css={`
+                    width: 100%;
+                  `}
+                >
+                  <FileUploaderField
+                    allowedMIMETypes={['image/jpeg', 'image/png']}
+                    file={formData[WRAPPABLE_LOGO]}
+                    id="file-uploader-3"
+                    label="ERC20 LOGO"
```

Comment:
```suggestion
                    label="EXISTING TOKEN ICON"
```

Reply:
I wonder to keep consistency between the two token assets names. We can call the one above: `GARDEN TOKEN ICON`.

Reply:
Already changed this 👌